cds/modules/deposit/api.py

```diff
@@ -221,6 +221,7 @@ def commit(self, **kwargs):
         """Set partial validator as default."""
         if 'validator' not in kwargs:
             kwargs['validator'] = PartialDraft4Validator
+        # TODO add 'modified_by' inside _cds
```

Comment:
I found the [issue](https://github.com/CERNDocumentServer/cds/issues/991) from Esteban! :smile: